README.md

```diff
@@ -2,64 +2,107 @@
+
+## Mission
+
+Crossplane strives to be the best Kubernetes add-on to provision and manage the
+infrastructure and services your applications need directly from kubectl. A
+huge part of this mission is arriving at an elegant, flexible way to define,
+compose, and publish your own infrastructure resources to the Kubernetes API
+and to model and manage cloud native applications.
+
+The path of cloud native apps from developer laptop into production requires
+collaboration across teams to build the app itself, deploy and manage the app
+and it’s infrastructure, and publishing infrastructure resources that embody
+organizational best practices and security policies.
+
+Today, multiple tools and management models must be glued together in
+deployment pipelines that are often fragile and error prone. Teams can find it
+difficult to collaborate in an effective way when aspects of an application are
+blurred, resulting in a lack of clear ownership and conflicts integrating
+changes. Requiring team members to master multiple tools, languages, and
+philosophies, while understanding the interactions and failure modes between
+them can significantly impede an organization’s ability to deliver applications
+efficiently.
+
+Crossplane believes that a team-centric approach with a strong separation of
+concerns combined with the proven Kubernetes declarative model is the best way
+to provision and manage infrastructure and cloud native applications. Teams
+should be able to publish infrastructure resources for applications to consume,
+define application components independent of infrastructure, and compose both
+into complete application configurations -- all using declarative YAML that can
+be deployed with kubectl from app delivery pipelines or with GitOps workflows.
+
+This team-centric approach reflects individuals often specializing in the
+following roles:
+
+*   **Infrastructure Operators** - provide infrastructure and services for apps
+    to consume
+*   **Application Developers** - build application components independent of
+    infrastructure
+*   **Application Operators** - compose, deploy, and run application
+    configurations
+
+This separation of concerns is core to Crossplane’s approach to infrastructure
+and application management, so team members can deliver value by focusing on
+what they know best.
+
+With Crossplane, infrastructure operators can define custom infrastructure
+resources with declarative YAML and publish them for applications to consume
+as Kubernetes custom resources or with any tool that works with the Kubernetes
+API. These infrastructure resources can be used with existing Kubernetes
+applications (Deployments, Services) and with application definition models
+like OAM.
+
+The result is a consistent, integrated, and modular approach to managing
+infrastructure and application configurations, that can be deployed with the
+same tooling including kubectl, GitOps, and anything can talk with the
+Kubernetes API.
```

Comment:
While you're updating the README, could you update "open an issue against Crossplane or the relevant stack" to "the relevant provider" down in the "Get Involved" heading?

Reply:
Done.